flyteidl/protos/flyteidl/admin/workflow.proto

```diff
@@ -90,3 +90,11 @@ message CreateWorkflowFailureReason {
         WorkflowErrorExistsIdenticalStructure exists_identical_structure = 2;
     }
 }
+
+message GetDynamicNodeWorkflowRequest {
```

Comment:
can we move this definition to  https://github.com/flyteorg/flyte/blob/master/flyteidl/protos/flyteidl/admin/node_execution.proto